### PR TITLE
Feat/game detail

### PR DESCRIPTION
**Pull request by Megavex**

### CODE REVIEW

File: application/app/src/main/java/com/yahyeet/boardbook/presenter/AccountManagerPresenter.java [hunk 1]
```diff
@@ -9,8 +9,15 @@
 
     private AccountManagerActivity accountManagerActivity;
 
+    // Set true to skip login
+    private Boolean fastPass = true;
```
Comment (cajoho99) [inline application/app/src/main/java/com/yahyeet/boardbook/presenter/AccountManagerPresenter.java]: This doesn't set a logged in user right?

Comment (Megavex) [reply application/app/src/main/java/com/yahyeet/boardbook/presenter/AccountManagerPresenter.java]: Yeah, only use if you want to test functionality that does not require a user and don't wanna log in every time

Comment (cajoho99) [reply application/app/src/main/java/com/yahyeet/boardbook/presenter/AccountManagerPresenter.java]: Could maybe set logged in user as a test user or something

Comment (Megavex) [reply application/app/src/main/java/com/yahyeet/boardbook/presenter/AccountManagerPresenter.java]: New user is now created if you use fastpass

**Pull request merged**